.circleci/config.yml

```diff
@@ -146,9 +146,14 @@ jobs:
       - setup_vm
       - install_chrome
       - run:
-          name: 'Visual Diff Tests'
-          command: node build-system/pr-check/visual-diff-tests.js
+          name: 'Visual Diff Tests (using << parameters.build_type >> build)'
+          command: node build-system/pr-check/visual-diff-tests.js --type << parameters.build_type >>
```

Comment:
I see that you're using `parameters:` with `build_type` enum with an explicit command-line flag called `--type` which is consumed by `visual-diff-tests.js`. Meanwhile, `Nomodule Tests` is using `matrix:` and `parameters:` with `process.env.config`.

I definitely think we should standardize how we parametrize test jobs and use a single style in both places. FWIW, I think the explicit parameter here is more readable than the `process.env` method.

Would it be too much to ask you to make the change here? (If it's too messy, we can clean up in a separate PR.)

/cc @danielrozenberg

Reply:
100% agree with you on that. Let me clean up `Module/Nomodule Tests` in a separate PR

Reply:
nope, willing to make the change! thanks for the recommendation

Reply:
@rsimha sorry. i meant yes, i am willing to make the change :D sorry if i worded that in a confusing way

Reply:
Oh crap, it was my fault! I understood you just fine, but used the wrong smiley. What a week it has been :)